doc/api/esm.md

```diff
@@ -1470,20 +1477,16 @@ export function getFormat(url, context, defaultGetFormat) {
   }
 
   // Let Node.js handle all other URLs.
-  return defaultGetFormat(url, context, defaultGetFormat);
+  return null;
 }
 
-export function transformSource(source, context, defaultTransformSource) {
-  const { url, format } = context;
-
+export function transformSource(source, { url }) {
   if (extensionsRegex.test(url)) {
     return {
-      source: CoffeeScript.compile(source, { bare: true })
+      source: CoffeeScript.compile(source.toString(), { bare: true })
```

Comment:
When would `source` not be a string? If a previous `getSource` returns a buffer?

Reply:
correct.